File: Python/uberPool.py
# Importamos las clases
from car import Car

# Clase UberPool (Herencia de la clase carro)
class UberPool (Car):

    # Atributos o propiedades de la clase
    brand = str
    model = str

    # Método constructor
    def __init__(self, license, driver, brand, model):
        super().__init__(license, driver)
        self.brand=brand
        self.model=model

File: Java/UberPool.java
//Clase UberPool (Hereda de la clase Carro)
class UberPool extends Car {
    // Atributos o propiedades de la clase
    String brand;
    String model;

    // Método constructor
    public UberPool(String license, Driver driver, String brand, String model) {
        super(license, driver);
        this.brand = brand;
        this.model = model;
    }
}